test/lib/testMayaUsdCreateStageCommands.py

```diff
@@ -59,7 +57,7 @@ def testCreateStageFromFile(self):
         # we can call what it does once the file is choose.
         # Note: on ballFilePath we replace \ with / to stop the \ as
         #       being interpreted.
-        ballFilePath = os.path.normpath(self.getTestScene('ballset', 'StandaloneScene', 'top_layer.usda')).replace('\\', '/')
+        ballFilePath = os.path.normpath(mayaUtils.getTestScene('ballset', 'StandaloneScene', 'top_layer.usda')).replace('\\', '/')
```

Comment:
Sorry, but what does this fix? I purposely copied the getTestScene() into this test to make it completely self contained. It doesn't rely or need the mayaUtils (which also imports ufe and ufeUtils). This test needs to run against previous versions of Maya.

Reply:
It fixes building/testing from a parent-project, where _maya-usd_ is included as a sub-project.
Lines 127-175 of _mayaUtils.py_ have no dependencies on **ufe**, so could easily be moved to _testUtils.py_

Reply:
Okay I understand now. I agree with moving those lines to a new _testUtils.py_ and then using that here instead of _mayaUtils_.